lib/Partitioner/Partitioner.cpp

```diff
@@ -328,6 +389,25 @@ void Partitioner::initOpComputeTime(Function *F) {
       totalOps *= (inputChannels * 1.0 / nGroups);
       break;
     }
+#ifdef GLOW_WITH_HABANA
+    case Kinded::Kind::HabanaConvolutionNodeKind: {
```

Comment:
This is copy pasted from above? Can it just share this logic?

Reply:
Yeah will look into that.

Reply:
I discussed offline; probably it makes most sense to couple this with the other restructuring changes in Partitioner.